package.json

```diff
@@ -113,6 +116,9 @@
     "karma-script-launcher": "~0.1.0",
     "less-plugin-remcalc": "^0.0.1",
     "lodash": "^3.0.1",
+    "npm-bower-sync-ver": "^1.1.2",
+    "npm-scripts-config": "^0.0.2",
+    "npm-scripts-conventional-changelog": "^0.1.0",
```

Comment:
I noticed here that you're using your own npm packages. Will they be migrated to the GitHub ovh-ux organization?

Reply:
Yes they should be migrated to ovh-ux organization.